### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by DerekRoy**
Hello, 

I am a Computing Science student in a technical writing class. I have been tasked with applying technical writing principles to technical documents. I found your repository while looking through similar repositories on github. The following changes are made to: promote clarity for readers using an active voice, eliminate ambiguity by removing pronouns like "it", and improve flow by rearranging sentence structure. Feel free to make changes to these suggestions to better suite your needs. I hope they are helpful. 

Thanks, 
Derek

### CODE REVIEW

**Comment by thelinuxfoundation** [comment]
Thank you for your pull request. Before we can look at your contribution, we need to ensure all contributors are covered by a Contributor License Agreement.

After the following items are addressed, please respond with a new comment here, and the automated system will re-verify.

* User @DerekRoy isn't covered by a CLA. They will need to complete the form at https://identity.linuxfoundation.org/projects/cncf

_Regards,
CLA GitHub bot_

**Comment by DerekRoy** [comment]
The CLA has been signed.

**Comment by mehrdada** [comment]
cc @srini100

File: README.md [hunk 1]
```diff
@@ -35,35 +35,35 @@ For instructions on how to use the language-specific gRPC runtime in your projec
...
+The "How to contribute" document also contains info on how the contribution process works and contains best practices for creating contributions.
 
 # Troubleshooting
 
 Sometimes things go wrong. Please check out the [Troubleshooting guide](TROUBLESHOOTING.md) if you are experiencing issues with gRPC.
 
 # Performance 
 
-See [Performance dashboard](http://performance-dot-grpc-testing.appspot.com/explore?dashboard=5636470266134528) for the performance numbers for the latest released version.
+See the [Performance dashboard](http://performance-dot-grpc-testing.appspot.com/explore?dashboard=5636470266134528) for performance numbers or the latest released version.
```
Comment (srini100) [inline README.md]: *of* the latest released version.

**Comment by grpc-testing** [comment]
```
Objective-C binary sizes
*****************STATIC******************
  New size                      Old size
 2,020,494      Total (=)      2,020,494

 No significant differences in binary sizes

***************FRAMEWORKS****************
  New size                      Old size
11,175,628      Total (>)     11,175,622

 No significant differences in binary sizes


```

File: README.md (hunk 1 above)
Comment (DerekRoy) [reply README.md]: Changed "or" to "of".

**Comment by DerekRoy** [comment]
Implemented changes.

**Comment by grpc-testing** [comment]
```
Objective-C binary sizes
*****************STATIC******************
  New size                      Old size
 2,020,494      Total (=)      2,020,494

 No significant differences in binary sizes

***************FRAMEWORKS****************
  New size                      Old size
11,175,631      Total (>)     11,175,630

 No significant differences in binary sizes


```

**Pull request merged**